### PR TITLE
Fix #15091- Import-AzWebAppKeyVaultCertificate api version bug

### LINKED ISSUE #15091: WebApp:Import-AzWebAppKeyVaultCertificate api version bug
## Description

When running the Import-AzWebAppKeyVaultCertificate command using the latest CLI and using a full pathed version of the keyvault "as its in a different subscription level", it returns the following response. 

_"The api-version '2016-10-01' is invalid. The supported versions are
'2021-04-01,2021-01-01,2020-10-01,2020-09-01,2020-08-01,2020-07-01,2020-06-01,2020-05-01,2020-01-01,2019-11-01,2019-10-
01,2019-09-01,2019-08-01,2019-07-01,2019-06-01,2019-05-10,2019-05-01,2019-03-01,2018-11-01,2018-09-01,2018-08-01,2018-0
7-01,2018-06-01,2018-05-01,2018-02-01,2018-01-01,2017-12-01,2017-08-01,2017-06-01,2017-05-10,2017-05-01,2017-03-01,2016
-09-01,2016-07-01,2016-06-01,2016-02-01,2015-11-01,2015-01-01,2014-04-01-preview,2014-04-01,2014-01-01,2013-03-01,2014-
02-26,2014-04'."_

While running the command using Debug mode, I can see that when its calling out to the keyvault API, its using the wrong API version to the one it used to get the list of keyvaults etc, the failing api version it uses is "2016-10-01" which is not in the accepted list for that API call. I suspect it might be a typo and should actually be "2016-09-01" which is what the prior API call uses before the one that is failing.

## Steps to reproduce

```powershell
$keyvaultPath = "/subscriptions/$($bindings.keyvault.subscriptionId)/resourceGroups/$($bindings.keyvault.resourceGroup)/providers/Microsoft.KeyVault/vaults/$($bindings.keyvault.name)"

Import-AzWebAppKeyVaultCertificate -ResourceGroupName $appBinding.resourceGroup -webAppName $appBinding.name -KeyVaultName  $keyvaultPath -CertName $certificateName -Debug
```

image
image

### PR DESCRIPTION
**Pull request by Kotasudhakarreddy**
#15091 
## Description
When running the Import-AzWebAppKeyVaultCertificate command using the latest CLI and using a full pathed version of the keyvault "as its in a different subscription level", it returns the following response.

"The api-version '2016-10-01' is invalid. The supported versions are
'2021-04-01,2021-01-01,2020-10-01,2020-09-01,2020-08-01,2020-07-01,2020-06-01,2020-05-01,2020-01-01,2019-11-01,2019-10-
01,2019-09-01,2019-08-01,2019-07-01,2019-06-01,2019-05-10,2019-05-01,2019-03-01,2018-11-01,2018-09-01,2018-08-01,2018-0
7-01,2018-06-01,2018-05-01,2018-02-01,2018-01-01,2017-12-01,2017-08-01,2017-06-01,2017-05-10,2017-05-01,2017-03-01,2016
-09-01,2016-07-01,2016-06-01,2016-02-01,2015-11-01,2015-01-01,2014-04-01-preview,2014-04-01,2014-01-01,2013-03-01,2014-
02-26,2014-04'

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by dingmeng-xue** [comment]
Azure PowerShell doesn't allow to depends on other service's management SDK because the limitation of assembly loading .NET core. Please share which operation of keyvault you need. Technically, we should upgrade common lib to support new version.

**Comment by Kotasudhakarreddy** [comment]
> Azure PowerShell doesn't allow to depends on other service's management SDK because the limitation of assembly loading .NET core. Please share which operation of keyvault you need. Technically, we should upgrade common lib to support new version.

I Need to make use KeyVault `GET` call. Yes, technically we should use common lib but the earlier version of code is used the specific version lib i.e. `Microsoft.Azure.Commands.Common.KeyVault.Version2016_10_1`. Now I have replaced everything with the common lib.

**Comment by isra-fel** [comment]
Hey, let's put this PR on hold until the discussion in #15091 reaches an agreement.

**Comment by Kotasudhakarreddy** [comment]
> Hey, let's put this PR on hold until the discussion in #15091 reaches an agreem

AKV SDK changes are rolled back. please review.

File: src/Websites/Websites/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* fixed `Import-AzWebAppKeyVaultCertificate` to support ServerFarmId.
```
Comment (BethanyZhou) [inline src/Websites/Websites/ChangeLog.md:21]:
```suggestion
* Fixed `Import-AzWebAppKeyVaultCertificate` to support ServerFarmId [#15091]
```

File: src/Websites/Websites/Utilities/KeyVaultClient.cs [hunk 2]
```diff
@@ -25,11 +25,21 @@ public KeyVaultManagementClient WrappedKeyVaultClient
             private set;
         }
 
-        public Vault GetKeyVault(string resourceGroupName, string vaultName)
+        public Vault GetKeyVault(string resourceGroupName, string vaultName, string kvSubscriptionId)
```
Comment (BethanyZhou) [inline src/Websites/Websites/Utilities/KeyVaultClient.cs:28]:
```suggestion
        public Vault GetKeyVault(string resourceGroupName, string vaultName, string kvSubscriptionId = null)
```
To keep compatibility with previous codes.

File: src/Websites/Websites/help/Import-AzWebAppKeyVaultCertificate.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ Accept wildcard characters: False
 ```
 
 ### -KeyVaultName
-The name of the keyvault.
+The name of the keyvault or Id of the KeyVault.
```
Comment (BethanyZhou) [inline src/Websites/Websites/help/Import-AzWebAppKeyVaultCertificate.md:76]: The description should be added in the definition of parameter `KeyVaultName` as well.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/Websites/Websites/ChangeLog.md (hunk 1 above)
Comment (Kotasudhakarreddy) [reply src/Websites/Websites/ChangeLog.md]: Done!

File: src/Websites/Websites/Utilities/KeyVaultClient.cs (hunk 2 above)
Comment (Kotasudhakarreddy) [reply src/Websites/Websites/Utilities/KeyVaultClient.cs]: Done!

**Review by BethanyZhou — APPROVED** [review]
LGTM

**Pull request merged**